src/ppom-controller.test.ts

```diff
@@ -263,6 +272,7 @@ describe('PPOMController', () => {
       }).rejects.toThrow('User has securityAlertsEnabled set to false');
     });
 
+    // TODO: Do we still need this test?
```

Comment:
We currently have constraint of mainnet in controller, but we will soon remove this.